src/index.ts

```diff
@@ -48,11 +95,12 @@ registerRecoveryRoute(app)
 registerRegistrationRoute(app)
 registerSettingsRoute(app)
 registerVerificationRoute(app)
+registerSessionsRoute(app)
 registerWelcomeRoute(app)
 registerErrorRoute(app)
 
 app.get("/", (req: Request, res: Response) => {
-  res.redirect("welcome", 303)
+  res.redirect(303, "sessions")
```

Comment:
This should redirect to `welcome`, I think?

Reply:
I don't think this should be to welcome since the hosted UI will directly be fed to customers as a "login" and "registration" screen. The welcome page is thus hidden from end-users and only developers launching it from the console will be directly redirected to the welcome page.

Reply:
This will have a massive impact on the Ory Proxy and Ory Tunnel and potentially the CNAME system. Please create an issue explaining what has changed so we can work on dealing with this in the Ory CLI.

Reply:
yeah I reverted this to `welcome` again. I did test it with the ory cli proxy and couldn't find a problem. But there might be some edge case I am missing. We can evaluate it later